### DISCUSSION TITLE
Starting from Zero

### DISCUSSION THREAD

**Discussion by Kuusho** [Cyfrin/foundry-full-course-cu#1006] [General]
I'm currently about to start lesson 6. I wanted to ask if anyone else started this course with no prior programming experience.

Also, is anyone learning a supplementary language like python or javascript to help wrap your head around some of the programming concepts?

**Comment by philiplorber** [comment]
Hello, I'm currently on lesson 7 and at the beginning had little to no experience programming. And yes, I've been learning suplementary languages like both of those, but also see a lot of videos and normally peek through some source code of projects to see how much I understand the code is doing. If you want you can add me on discord: philipo0 Good luck on your journey!

**Reply by Kuusho** [reply]
This is awesome dude! I'll add you up for sure, Thanks for responding.

**Reply by philiplorber** [reply]
If you happen to add me tell me your disc name here, since I get a lot of bots / spam requests! Again gl with the coding

**Reply by Kuusho** [reply]
kuusho. 

same as my discord

**Reply by philiplorber** [reply]
bro it is me philipo lol not a scammer

**Reply by Kuusho** [reply]
GM Philipo, just sent you a request on discord!

**Comment by JC-mb33** [comment]
Same here. Good look you both!

**Reply by Kuusho** [reply]
Goodluck to you too fren <3

**Comment by msrusskikh** [comment]
Hey guys! I'm also a complete newbie in programming, currently at lesson 2. I've started a complimentary course on JS just because it is kind of a prerequisite to solidity on Alchemy bootcamp, but don't think you may need a ton of knowledge about other langs to be proficient in Solidity. Good luck to you all)
